exe/wallet/Main.hs

```diff
@@ -113,6 +121,7 @@ Options:
   --bridge-port <INT>         port used for communicating with the http-bridge [default: 8080]
   --address-pool-gap <INT>    number of unused consecutive addresses to keep track of [default: 20]
   --size <INT>                number of mnemonic words to generate [default: 15]
+  --target <STRING>           address to send to and amount to send separated by @: '<amount>@<address>'
```

Comment:
I feel that `target` is perhaps not the right name here, as it's really a (target, amount) pair.

Perhaps we can think of a better name? For example, how about `payment`, which combines a recipient (target) and an amount.

Reply:
This matches the name we have in the API spec. If we change it here, we should change it everywhere too.

Reply:
I see the importance of keeping everything in sync.

In that case, I think the name in the API spec is a bit confusing. My reasoning is that the word "target" already has a strong connotation of being just a target, rather than the thing that one sends to a target. Perhaps we can rename this to "payment" or "target-amount".

Reply:
@KtorZ @akegalj @rvl Out of the choices of:

* `--payment`
* `--amount-recipient` (where the value is specified as `amount@recipient`)
* `--recipient-amount` (where the value is specified as `recipient:amount`)

Which would people prefer?

Reply:
Not the latter, as the recipient is an address that is rather "long" compared to the amount. I think it makes more sense to put the amount first for the sake of readability. I like `payment` for it's rather concise and clear.

Reply:
fixed